<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>actionDefinitionWidget</class>
 <widget class="QWidget" name="actionDefinitionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>358</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>actionDefinitionWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QGroupBox" name="m_ProcessParameters">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Właściwości procesu</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Ip maszyny:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Parametry:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="m_AddParameter">
          <property name="text">
           <string>Dodaj</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_RemoveParameter">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Usuń</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QTableWidget" name="m_ParameterList">
        <property name="minimumSize">
         <size>
          <width>302</width>
          <height>0</height>
         </size>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>150</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Nazwa</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Wartość</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="m_MachineIp">
        <property name="text">
         <string>0.0.0.0</string>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="m_Processes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Procesy</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_1">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item row="0" column="1">
       <widget class="QPushButton" name="m_AddProcess">
        <property name="text">
         <string>Dodaj</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="m_RemoveProcess">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Usuń</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QListWidget" name="m_ProcessList">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="uniformItemSizes">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="m_DuplicateProcess">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Duplikuj</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>m_AddProcess</tabstop>
  <tabstop>m_RemoveProcess</tabstop>
  <tabstop>m_ProcessList</tabstop>
  <tabstop>m_ParameterList</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_AddProcess</sender>
   <signal>clicked()</signal>
   <receiver>actionDefinitionWidget</receiver>
   <slot>addProcess()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_RemoveProcess</sender>
   <signal>clicked()</signal>
   <receiver>actionDefinitionWidget</receiver>
   <slot>removeProcess()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_ProcessList</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>actionDefinitionWidget</receiver>
   <slot>currentProcessChanged(QListWidgetItem*,QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_AddParameter</sender>
   <signal>clicked()</signal>
   <receiver>actionDefinitionWidget</receiver>
   <slot>addParameter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_RemoveParameter</sender>
   <signal>clicked()</signal>
   <receiver>actionDefinitionWidget</receiver>
   <slot>removeParameter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_ParameterList</sender>
   <signal>cellChanged(int,int)</signal>
   <receiver>actionDefinitionWidget</receiver>
   <slot>parameterChanged(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_MachineIp</sender>
   <signal>editingFinished()</signal>
   <receiver>actionDefinitionWidget</receiver>
   <slot>machineIpChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_ParameterList</sender>
   <signal>currentItemChanged(QTableWidgetItem*,QTableWidgetItem*)</signal>
   <receiver>actionDefinitionWidget</receiver>
   <slot>currentParameterChanged(QTableWidgetItem*,QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_DuplicateProcess</sender>
   <signal>clicked()</signal>
   <receiver>actionDefinitionWidget</receiver>
   <slot>duplicateProcess()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addProcess()</slot>
  <slot>removeProcess()</slot>
  <slot>currentProcessChanged(QListWidgetItem*,QListWidgetItem*)</slot>
  <slot>addParameter()</slot>
  <slot>removeParameter()</slot>
  <slot>parameterChanged(int,int)</slot>
  <slot>machineIpChanged()</slot>
  <slot>currentParameterChanged(QTableWidgetItem*,QTableWidgetItem*)</slot>
  <slot>duplicateProcess()</slot>
 </slots>
</ui>
